shell-completion/zsh/_machinectl

```diff
@@ -4,34 +4,41 @@
 {
   local -a _machinectl_cmds
   _machinectl_cmds=(
-    "list:List currently running VMs/containers"
-    "status:Show VM/container status"
-    "show:Show properties of one or more VMs/containers"
-    "start:Start container as a service"
-    "stop:Stop container (equal to poweroff)"
-    "login:Get a login prompt on a VM/container"
-    "enable:Enable automatic container start at boot"
-    "disable:Disable automatic container start at boot"
-    "poweroff:Power off one or more VMs/containers"
-    "reboot:Reboot one or more VMs/containers"
-    "terminate:Terminate one or more VMs/containers"
-    "kill:Send signal to process or a VM/container"
-    "copy-to:Copy files from the host to a container"
-    "copy-from:Copy files from a container to the host"
-    "bind:Bind mount a path from the host into a container"
+    "list:list currently running VMs/containers"
+    "status:show VM/container status"
+    "show:show properties of one or more VMs/containers"
+    "start:start container as a service"
+    "stop:stop container (equal to poweroff)"
+    "login:get a login prompt on a VM/container"
+    "shell:open an interactive shell in a container"
+    "enable:enable automatic container start at boot"
+    "disable:disable automatic container start at boot"
+    "poweroff:power off one or more VMs/containers"
+    "reboot:reboot one or more VMs/containers"
+    "terminate:terminate one or more VMs/containers"
+    "kill:send signal to process or a VM/container"
+    "copy-to:copy files from the host to a container"
+    "copy-from:copy files from a container to the host"
+    "bind:bind mount a path from the host into a container"
```

Comment:
This too should be two commits (at least). One for the trivial case changes, other for the changes in logic.